hail/python/hail/expr/functions.py

```diff
@@ -1753,6 +1753,95 @@ def pnorm(x) -> Float64Expression:
     return _func("pnorm", tfloat64, x)
 
 
+@typecheck(x=expr_float64, n=expr_float64, lower_tail=expr_bool, log_p=expr_bool)
+def pT(x, n, lower_tail=True, log_p=False) -> Float64Expression:
```

Comment:
nice! this will make TJ happy :)

Reply:
these functions need tests, though (I think our policy is that doctests don't philosophically count as correctness tests). Can just be duplicates of the doctests you have here, but in `test_expr.py`